assignments/this.js

```diff
@@ -1,10 +1,18 @@
-/* The for principles of "this";
+/* The four principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Whenever a function is contained in the global scope, the
+value of ${this} inside that function will be the window object.
+
+* 2. Implicit binding occurs when dot notation is used to invoke a function
+
+* 3. Constructor functions return an object. A constructor function 
+is used in object oriented programming with the keyword "new" in New Binding
```

Comment:
Here, you're talking about how constructor functions make new objects with the `new` keyword. The key thing here is that `this` gets correctly set to the newly created object when the properties are being set.